### ISSUE TITLE
Can't Add Punctuation To Community Description

### ISSUE THREAD

**Issue by Samyoul** [status-im/status-app#12701]
# Bug Report

## Description

As a user I expect to be able to enter punctuation so that I can write grammatically correct descriptions.

<img width="654" alt="Screenshot 2023-11-13 at 10 46 19" src="https://github.com/status-im/status-desktop/assets/5702426/a06849ae-611d-456a-a020-94e067ee9f8b">

This is probably not a bug, but it feels like a strange restriction 🤷 

### Additional Information

- Status desktop version: 0.14.1
- Operating System: MacOS Ventura 13.5.2

**Comment by caybro** [comment]
Alright, this is a "feature", not a bug :)

**Comment by noeliaSD** [comment]
@benjthayer WDYT?

**Comment by caybro** [comment]
It was decided against using anything else than what's described in the red error msg

**Comment by benjthayer** [comment]
> @benjthayer WDYT?

Yeah I'm not 100% clear on the rationale for the restriction here. Is it to do with how this info is included in the URL do you know @xAlisher ?

**Comment by noeliaSD** [comment]
cc: @xAlisher ??

**Comment by xAlisher** [comment]
Let's allow punctuation, no reason for restrictions here.

**Comment by jrainville** [comment]
Now fixed
